tests/endtoend/test_worker_proccess_count_functions.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import os
+from threading import Thread
+from unittest.mock import patch
+from datetime import datetime
+from tests.utils import testutils
+
+
+class TestWorkerProcessCount1(testutils.WebHostTestCase):
```

Comment:
I don't think we need this test cases. The below testcase of setting fwpc to 2 should be enough to test this feature